Package.swift

```diff
@@ -99,55 +99,59 @@ let package = Package(
 
         // MARK: SwiftPM specific support libraries
 
+        .target(
+            name: "Core",
+            dependencies: ["SwiftToolsSupport-auto"]),
+
         .target(
             /** The llbuild manifest model */
             name: "LLBuildManifest",
-            dependencies: ["SwiftToolsSupport-auto"]),
```

Comment:
Any module that already imports modules from ToolSupportCore should probably add the new dependency in addition to the TSC one, not instead of it.  I think eventually we want to sandbox things so that everything declares its actual dependencies, so this should reflect the imports actually being used by the module.

Reply:
okay